website/docs/components/flyout/partials/guidelines/guidelines.md

```diff
@@ -114,11 +114,11 @@ The Flyout header features several properties to better communicate the purpose
 
 **With title icon**
 
-<Hds::Flyout::Header @icon="info" @onDismiss={{this.noop}}>Title</Hds::Flyout::Header>
+![Flyout header title with icon and dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-icon-and-title.jpg)
 
 **Without title icon**
 
-<Hds::Flyout::Header @onDismiss={{this.noop}}>Title</Hds::Flyout::Header>
+![Flyout header title only and dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-title-only.jpg)
```

Comment:
```suggestion
![Flyout header title with just dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-title-only.jpg)
```

Reply:
I think the meaning is different here. 🤔

Reply:
The image is of a Flyout header with only a dismiss button and not an icon. So you could change the text to be more along those lines. (It's not a "flyout header title only" since it also has a dismiss button.)

Reply:
Leaving the original version, so resolving this now.